sdk/keyvault/keyvault-keys/test/public/crypto.spec.ts

```diff
@@ -83,6 +84,18 @@ describe("CryptographyClient (all decrypts happen remotely)", () => {
       const decryptedText = uint8ArrayToString(decryptResult.result);
       assert.equal(text, decryptedText);
     });
+
+    it("the CryptographyClient can be created from a full KeyVaultKey object", async function() {
+      const keyName = testClient.formatName(`${keyPrefix}-${this!.test!.title}-${keySuffix}`);
+      const keyVaultKey = await client.createKey(keyName, "RSA");
```

Comment:
Other tests in this block use the key created in the `beforeEach()` block. Any reason why this test deviates from that pattern?

Reply:
I wanted to show that a new key with nothing special can be used to create this client.